CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Updated semver transitive dependency to resolve a potential vulnerability.
```

Comment:
I think we often try to keepthe changelogs vague and say something along the lines of `Update dependency XYZ for technical currency` 😋 

Not a big deal though